rpc/client/peer-rpc-clnt.go

```diff
@@ -49,3 +49,34 @@ func ValidateAddPeer(p *peer.PeerAddRequest) (*services.RPCPeerAddResp, error) {
 	}
 	return rsp, nil
 }
+
+func ExportEtcdEnvVar(argv *services.RPCEtcdEnvReq) (*services.RPCEtcdEnvResp, error) {
```

Comment:
This actually should be Add() function as we'd need to communicate additional information related to etcd env variables but in the add flow.